.github/workflows/build_images.yaml

```diff
@@ -36,17 +36,27 @@ jobs:
 
       - name: Login to Harbor, if building master
         uses: docker/login-action@v2
-        if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
Why was this removed? It seems to contradict the description.

Reply:
So we always login, then we either push to staging or prod based on the branch
Since the login happens down both paths we could remove this check

Reply:
Is it worth getting rid of `if building master` in the name then?

Reply:
Done :+1: